pkgs/applications/misc/glogg/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, qt5, boost }:
+
+stdenv.mkDerivation rec {
+
+  name = "glogg-1.1.4";
+  
+  src = fetchurl {
+    url = "http://glogg.bonnefon.org/files/${name}.tar.gz";
+    sha256 = "0c1ddc72ebfc255bbb246446fb7be5b0fd1bb1594c70045c3e537cb6d274965b";
+  };
+
+  buildInputs = [ qt5.qmake boost ];
+
+  qmakeFlags = [ "glogg.pro" ];
+
+  installCommand = ''
+    make install INSTALL_ROOT=$out
```

Comment:
I don't think this will be run. ```installCommand``` is not used.

Reply:
Yes that's right, I can delete it and it will work. Will update my PR.